src/Tests/Microsoft.NET.Sdk.Razor.Tests/RazorSdkTest.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.NET.TestFramework.Commands;
```

Comment:
License header.

Reply:
You might also not need a bunch of these namespaces.

Reply:
Fixed.